isobus/include/isobus/isobus/isobus_virtual_terminal_client.hpp

```diff
@@ -293,6 +331,65 @@ namespace isobus
+
+		/// @brief Allows you to register for a callback when a numeric value is changed in an input object
+		/// @param[in] value The callback to register
+		void RegisterVTChangeNumericValueEventCallback(VTChangeNumericValueCallback value);
+
+		/// @brief Allows you to remove a numeric value change callback
+		/// @param[in] value The callback to remove
+		void RemoveVTChangeNumericValueEventCallback(VTChangeNumericValueCallback value);
+
+		/// @brief Allows you to register for a callback when the active mask is changed
+		/// @details The VT sends this whenever there are missing object references or errors in the mask.
+		/// @param[in] value The callback to register
+		void RegisterVTChangeActiveMaskEventCallback(VTChangeActiveMaskCallback value);
+
+		/// @brief Allows you to remove a callback when the active mask is changed
+		/// @param[in] value The callback to remove
+		void RemoveVTChangeActiveMaskEventCallback(VTChangeActiveMaskCallback value);
+
+		/// @brief Allows you to register for a callback when the soft key mask is changed
+		/// @details The VT sends this whenever there are missing object references or errors in the mask.
+		/// @param[in] value The callback to register
+		void RegisterVTChangeSoftKeyMaskEventCallback(VTChangeSoftKeyMaskCallback value);
+
+		/// @brief Allows you to remove a callback when the soft key mask is changed
+		/// @param[in] value The callback to remove
+		void RemoveVTChangeSoftKeyMaskEventCallback(VTChangeSoftKeyMaskCallback value);
+
+		/// @brief Allows you to register for a callback when a string value is changed
+		/// @details The object could be either the input string object or the referenced string variable object.
+		/// @param[in] value The callback to register
+		void RegisterVTChangeStringValueEventCallback(VTChangeStringValueCallback value);
+
+		/// @brief Allows you to remove a callback when a string value is changed
+		/// @param[in] value The callback to remove
+		void RemoveVTChangeStringValueEventCallback(VTChangeStringValueCallback value);
+
+		/// @brief Allows you to register for a callback when a user-layout object is hidden or shown
+		/// @param[in] value The callback to register
+		void RegisterVTUserLayoutHideShowEventCallback(VTUserLayoutHideShowCallback value);
+
+		/// @brief Allows you to remove a callback when a user-layout object is hidden or shown
+		/// @param[in] value The callback to remove
+		void RemoveVTUserLayoutHideShowEventCallback(VTUserLayoutHideShowCallback value);
+
+		/// @brief Allows you to register for a callback when an audio signal is terminated
+		/// @param[in] value The callback to register
+		void RegisterVTControlAudioSignalTerminationEventCallback(VTAudioSignalTerminationCallback value);
+
+		/// @brief Allows you to remove a callback when an audio signal is terminated
+		/// @param[in] value The callback to remove
+		void RemoveVTControlAudioSignalTerminationEventCallback(VTAudioSignalTerminationCallback value);
```

Comment:
Wondering if UpperCamelCase naming is desired for these? I followed the style of the ones that were already present

Reply:
I've been trying (and apparently not always succeeding) to do `snake_case` for functions. I should really set up clang-tidy checking for this at some point. So I suppose in my perfect world they'd be `snake_case` kind of like other places, like `get_connected_vt_version` etc